tools/scripts/examples_compile.py

```diff
@@ -69,16 +68,24 @@ def build(project):
 
 def run(project):
 	path = project.parent
-	if is_running_in_ci:
-		command = "scons run"
-	else:
-		command = "python3 `which scons` run"
-	output = ["=" * 90, "Running: {} with {}".format(path / "main.cpp", "SCons" if "scons" in command else "CMake")]
-	rc, ro = run_command(path, command, all_output=True)
-	print("\n".join(output + [ro]))
-	if "CI: run fail" in project.read_text():
-		return None if not rc else project
-	return None if rc else project
+	project_cfg = project.read_text()
+	commands = []
+	if ":build:scons" in project_cfg:
+		commands.append( ("scons run", "SCons") )
+	if ":build:make" in project_cfg:
+		commands.append( ("make run", "Make") )
+
```

Comment:
What about cmake?

Reply:
Eh, let's fix it in #568. I'd have to add a `make run` to the Makefile that will be removed…